users/api.py

```diff
@@ -206,3 +217,29 @@ def find_available_username(initial_username_base):
         )
         # If there is space for 4 digits for the suffix, the minimum value it could be is 1000, or 10^3
         current_min_suffix = 10 ** (available_suffix_digits - 1)
+
+
+def update_edx_user(user):
```

Comment:
1. While this is user-related functionality, this function should be defined in `courseware/api.py` and not `users/api.py` since it's interacting with our courseware (edX). The constant should be defined in `courseware/constants.py`
1. A more accurate name for this function would be `update_edx_user_name`
1. It's fine to construct the API call in this way for now, but this functionality should really be defined in the [edx-api-client](https://github.com/mitodl/edx-api-client). Can you please create an issue on that repo to implement this, and also a tech debt issue in this repo to refactor this logic to use edx-api-client to perform the update request instead?

Reply:
The above changes have been incorporated and created suggested relevant issues as well.